src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,14 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private final int height;
+
     public Excavator() {
+        height = 15;
```

Comment:
you should leave the no-args constructor untouched, but create another one with the height field and initialize it in the place you use it, not hardcoding the value here